corehq/apps/cloudcare/tests/test_api.py

```diff
@@ -304,13 +303,13 @@ def _create_case(user, type, close=False, **extras):
         user_id=user.user_id,
         owner_id=user.user_id,
         **extras
-    ).as_xml(format_datetime=date_to_xml_string)]
```

Comment:
i think it actually has the same functionality. date_to_xml_string just calls json_format_date which is the default `format_date` function for `as_xml` with the one difference being that `date_to_xml_string` allows for `None` dates